people/swapnabio.md

```diff
@@ -0,0 +1,18 @@
+<h1 align="center">Hi 👋, I'm Swapna Chemboth. Currently living in Brussels,Belgium</h1>
```

Comment:
Hi @SWAPNACHEMBOTH 
You can replace all the h1, h2.. with markdown equivalents such as '# heading' for h1